pom.xml

```diff
@@ -964,27 +964,6 @@
             </dependency>
 
             <!-- Hudi dependency  -->
-            <!--
-              We don't use hadoop-common directly, it's only for suppressing exception:
-                 Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.2.4:shade (default) on project
-                 kyuubi-spark-sql-engine_2.12: Error creating shaded jar: Could not resolve following dependencies:
-                 [jdk.tools:jdk.tools:jar:1.6 (system)]
-
-              The issue only occurs on GitHub Action environment with Hudi 0.10.0 and JDK 11.
-              After few days digging, only found one place introduces jdk.tools,
-
-              - org.apache.hudi:hudi-common:jar:0.10.0:test
-                - org.apache.hbase:hbase-server:jar:1.2.3:test
-                  - org.apache.hadoop:hadoop-common:jar:2.5.1:test
-                    - org.apache.hadoop:hadoop-annotations:jar:2.5.1:test
-                      - jdk.tools:jdk.tools:jar:1.6:system
-            -->
-            <dependency>
-                <groupId>org.apache.hadoop</groupId>
-                <artifactId>hadoop-common</artifactId>
```

Comment:
[HUDI-1180](https://issues.apache.org/jira/browse/HUDI-1180) Upgrade HBase from 1.2.3 to 2.4.9.
[HBASE-18640](https://issues.apache.org/jira/browse/HBASE-18640) Fixes leaks `jdk.tools` dependency in version 2.x.
So I try to remove `hadoop-common` dependency.

Reply:
Hudi 0.11 has a different module dependency graph, the exclusion may need to update. 

BTW, seems that Hudi 0.11 reduced the bundle jar dependencies, but I failed to pass the ut after switching to bundle jar w/ spark 3.2 in my previous try.

Reply:
I first tried to pass the existing UT based on Spark3.1, so I changed hudi-spark3 to hudi-spark3.1.x, because hudi-spark3 uses spark3.2.

The next step is to plan to support hudi integration test in Spark 3.2.

I'll check for redundant exclusions later.